### ISSUE TITLE
Translating Seeing Theory Website: HowTo

### ISSUE THREAD

**Issue by jdittrich** [seeingtheory/Seeing-Theory#29]
How can I translate the seeing theory webpage (like existing ones in [*es*](https://seeing-theory.brown.edu/es.html) and [*cn*](https://seeing-theory.brown.edu/cn.html))?

**User Story:** As a learner of statistics, I would like to read and *seeing theory* in the language I speak best
**Contributor Story:** As a speaker of both English and an other language, I would like to translate *seeing theory*

**Comment by SilverLeaves** [comment]
hahahaha... you are programer, just do what you want to do... you are funny
1. install a text editor, like notepad、atom、[visual code](https://code.visualstudio.com/) etc.
2. fork this repository
3. copy a *.html
4. translation
5. fix link

if you want to preview, run a local server, python 、node or any one else http server etc.
hahahaha... you are programer, just do what you want to do... you are funny

**Comment by ThomDietrich** [comment]
I believe this ticket is directed at a way of making this project "localizable". This is the only sane way to support mass-translation while it still allows for improvements to the extend of the course, the page style & structure, and so on.

A typical way of doing so is to give every string a unique identifier, then exporting them to a language resource file (xliff, xml, ini, ...), then using a framework or piece of logic to present the correct translation to the end user.
There are a few solutions for static html pages out there and a few platforms ease the mass-translation of said resource file. I personally made good experience with https://crowdin.com

**Comment by ThomDietrich** [comment]
Fyi crowdin also supports html files directly. This might not be the most elegant solution but would be the easiest if @danielkunin and @jingruguo or another willing contributor don't want to implement a decoupled method.

https://support.crowdin.com/supported-formats

**Comment by jingruguo** [comment]
Thanks Thomas! It’s a great suggestion and I will look into it.

-Jingru

**Comment by SilverLeaves** [comment]
@ThomDietrich Have you seen the code of this project?

This is a completely static project. The project developer has spent a lot of energy developing it with the old JQ1.1.

For the time being, it's hard to support what you're saying.

In the long run, the project needs to change to a new modern engineering architecture, such as Vue (with vue-i18n), React (with react-intl), to support internationalization and large-scale page development.

**Comment by ThomDietrich** [comment]
True. That is why I mentioned but discouraged the second option (crowdin support for html) 🤷‍♂
